spec.html

```diff
@@ -13693,7 +13691,21 @@ <h1>
                 1. Let _initialValue_ be ! _env_.GetBindingValue(_n_, *false*).
               1. Perform ! _varEnv_.InitializeBinding(_n_, _initialValue_).
               1. NOTE: A var with the same name as a formal parameter initially has the same value as the corresponding initialized parameter.
-        1. [id="step-functiondeclarationinstantiation-web-compat-insertion-point"] NOTE: Annex <emu-xref href="#sec-web-compat-functiondeclarationinstantiation"></emu-xref> adds additional steps at this point.
+        1. [id="step-functiondeclarationinstantiation-web-compat-insertion-point", normative-optional] If the host is a web browser or otherwise supports <emu-xref href="#sec-block-level-function-declarations-web-legacy-compatibility-semantics" title></emu-xref> and _strict_ is *false*, then
```

Comment:
There's a tiny risk of ambiguity with where the `and _strict_ is *false*` binds, but I think it's probably fine.

Reply:
I factored out `If _strict_ is *false*`, then merged it with the subsequent step, then swapped the sense of the `If` to put the short arm first.

In the similar case in EvalDeclarationInstantiation, I swapped the order of the terms in the conjunction (putting `_strict_ is *false*` first, host-condition last), which I think is enough to prevent misreading.

Likewise, in the 4 early error rules (mentioned [above](https://github.com/tc39/ecma262/pull/2952#issuecomment-1309699433)) of the form:
> It is a Syntax Error if [cond1], unless the host is a web browser or otherwise supports [feature], and [cond2].

I changed them to put the host-condition last:
> It is a Syntax Error if [cond1], unless [cond2] and the host is a web browser or otherwise supports [feature].